### PR TITLE
restack

### PR DESCRIPTION
**Pull request by ahdinosaur**
part of https://github.com/root-systems/dogstack/pull/91

### CODE REVIEW

File: app/components/Navigation.js [hunk 1]
```diff
@@ -29,59 +35,70 @@ function Navigation (props) {
...
+            onClick={toggleDrawer}
+            button={true}
             leftIcon={
               <i className={icon} aria-hidden="true" />
             }
           />
         )
+      } else {
+        item = (
+          <NavLink to={path} key={name}>
```
Comment (ahdinosaur) [inline app/components/Navigation.js]: since we're touching this file, should we `h()`-ify it?

File: app/themes/base.js [hunk 2]
```diff
@@ -1,24 +1,4 @@
...
-  grey500,
-  grey600,
-  grey700,
-  grey800,
-  grey900,
-  darkBlack,
-  fullBlack,
-  white
-} from 'material-ui/styles/colors'
+import {green, purple, grey, black, white} from '@material-ui/core/colors'
```
Comment (ahdinosaur) [inline app/themes/base.js:1]: this is unrelated / maybe related: apparently material-ui as a much improved theme structure, we might want to copy them more closely. @mischa-s was saying something about this?

File: config/production.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
 module.exports = {
-  port: process.env.PORT,
-  app: {
-    url: process.env.APP_URL
+  api: {
+    port: process.env.PORT,
+    name: process.env.APP_NAME,
```
Comment (ahdinosaur) [inline config/production.js]: the idea here is the ENV_NAME should always match the config path, so if you want `APP_NAME` it should either correspond to `app.name` or should be `API_NAME`.

Comment (ahdinosaur) [reply config/production.js]: from recollection, this is the app name, not to do specifically with the api.

File: browser.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+const client = {
+  services: [
+    authentication
+  ],
+  apiUrl: config.api.url
+}
+
+// root
+const appNode = document.createElement('div')
```
Comment (ahdinosaur) [inline browser.js]: could this go in the `index.html`? i think i was the one who did this, no idea why, probably to move quickly and break things.

File: netlify.toml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[build]
```
Comment (ahdinosaur) [inline netlify.toml:1]: have you included the redirects file?


https://www.netlify.com/docs/redirects/

we need to handle pushState routes.

**Review by ahdinosaur — COMMENTED** [review]
wubble dupple bub bub :sparkle:

File: netlify.toml (hunk 5 above)
Comment (ahdinosaur) [reply netlify.toml:1]: https://github.com/peachcloud/peachhub-provider/blob/master/dist/_redirects

File: app/themes/base.js (hunk 2 above)
Comment (iainkirkpatrick) [reply app/themes/base.js:1]: good thought - i'll break this out into another ticket i reckon and tackle it that way :)

File: app/components/Navigation.js (hunk 1 above)
Comment (iainkirkpatrick) [reply app/components/Navigation.js]: good call :) have done!

File: config/production.js (hunk 3 above)
Comment (iainkirkpatrick) [reply config/production.js]: fixed

File: netlify.toml (hunk 5 above)
Comment (iainkirkpatrick) [reply netlify.toml:1]: included in the `netlify.toml` file
